README.md

```diff
@@ -5,6 +5,27 @@
 
 # che-workspace-telemetry-woopra-plugin
 
+## Running the Devfile
```

Comment:
I think it worth removing also CI badges from the readme header:
```
[![Master Build Status](https://ci.centos.org/buildStatus/icon?subject=master&job=devtools-che-workspace-telemetry-woopra-plugin-build-master/)](https://ci.centos.org/view/Devtools/job/devtools-che-workspace-telemetry-woopra-plugin-build-master/)
[![Nightly Build Status](https://ci.centos.org/buildStatus/icon?subject=nightly&job=devtools-che-workspace-telemetry-woopra-plugin-che-nightly/)](https://ci.centos.org/view/Devtools/job/devtools-che-workspace-telemetry-woopra-plugin-che-nightly/)
```

Reply:
Removed the badges